native/cocos/bindings/manual/jsb_spine_manual.cpp

```diff
@@ -213,6 +214,314 @@ static bool js_register_spine_retainSkeletonData(se::State &s) {
+        args[1].toObject()->setProperty("y", se::Value(sizey));
+        if (!temp.isNullOrUndefined()) {
+            for (int i = 0; i < outVertexBuffer.size(); ++i) {
+                temp.toObject()->setArrayElement(i, se::Value(outVertexBuffer[i]));
+            }
+        }
+    }
+    return true;
+}
+SE_BIND_FUNC(js_getBounds_Skeleton)
+
+static bool js_worldToLocal_Bone(se::State &s) {
+    const auto &args = s.args();
+    spine::Bone* bone = SE_THIS_OBJECT<spine::Bone>(s);
+    if (nullptr == bone) return true;
+
+    spine::Vector2 world(0, 0);
+
+    bool ok = false;
+    ok = sevalue_to_native(args[0], &world, s.thisObject());
+
+    float outX, outY;
+    bone->worldToLocal(world.x, world.y, outX, outY);
+
+    spine::Vector2 outNative(outX, outY);
+    se::Value ret;
+    nativevalue_to_se(outNative, ret, s.thisObject());
+    s.rval().setObject(ret.toObject());
+    return true;
+}
+SE_BIND_FUNC(js_worldToLocal_Bone)
+
+static bool js_localToWorld_Bone(se::State &s) {
+    const auto &args = s.args();
+    spine::Bone* bone = SE_THIS_OBJECT<spine::Bone>(s);
+    if (nullptr == bone) return true;
+
+    spine::Vector2 local(0, 0);
+
+    bool ok = false;
+    ok = sevalue_to_native(args[0], &local, s.thisObject());
+
+    float outX, outY;
+    bone->localToWorld(local.x, local.y, outX, outY);
+
+    spine::Vector2 outNative(outX, outY);
+    se::Value ret;
+    nativevalue_to_se(outNative, ret, s.thisObject());
+    s.rval().setObject(ret.toObject());
+    return true;
+}
+SE_BIND_FUNC(js_localToWorld_Bone)
+
+static bool js_computeWorldPosition_PointAttachment(se::State &s) {
+    const auto &args = s.args();
+    spine::PointAttachment* pointAttachment = SE_THIS_OBJECT<spine::PointAttachment>(s);
+    if (nullptr == pointAttachment) return true;
+
+    spine::Bone* bone = nullptr;
+
+    bool ok = false;
+    ok = sevalue_to_native(args[0], &bone, s.thisObject());
+
+    float outX, outY;
+    pointAttachment->computeWorldPosition(*bone, outX, outY);
+
+    spine::Vector2 outNative(outX, outY);
+    se::Value ret;
+    nativevalue_to_se(outNative, ret, s.thisObject());
+    s.rval().setObject(ret.toObject());
+    return true;
+}
+SE_BIND_FUNC(js_computeWorldPosition_PointAttachment)
+
+static bool js_findAttachmentsForSlot_Skin(se::State &s) {
+    const auto &args = s.args();
+    spine::Skin* skin = SE_THIS_OBJECT<spine::Skin>(s);
+    if (nullptr == skin) return true;
+
+    size_t slotIndex = 0;
+    se::Value attachmentsVal;
+
+    bool ok = false;
+    ok = sevalue_to_native(args[0], &slotIndex, s.thisObject());
+    ok = sevalue_to_native(args[1], &attachmentsVal, s.thisObject());
+
+    spine::Skin::AttachmentMap::Entries entries = skin->getAttachments();
+    int index = 0;
+    while (entries.hasNext()) {
+        spine::Skin::AttachmentMap::Entry &entry = entries.next();
+        if (entry._slotIndex == slotIndex) {
+            attachmentsVal.toObject()->setArrayElement(index++, se::Value(&entry));
```

Comment:
setArrayElement accesses `uint32_t` as its first argument. So `int index = 0;` needs to be changed to `uint32_t index = 0;`

Reply:
And what does `se::Value(&entry)` stand for? If you want to convert a native pointer to se::Value, use se `nativevalue_to_se`